packages/react-components/src/index.ts

```diff
@@ -12,7 +12,7 @@ export { default as AddressSmall } from './AddressSmall.js';
 export { default as AddressToggle } from './AddressToggle.js';
 export { default as Available } from './Available.js';
 export { default as AvatarItem } from './AvatarItem.js';
-export { default as AzeroId, AzeroIdDomain } from './AzeroId.js';
+export { default as AzeroId, AzeroIdDomain } from './AzeroId/index.js';
```

Comment:
That's rather not necessary?

Reply:
I don't think it works without it :cry:  it's some weird polkadot config quirk I guess